Library/Homebrew/utils/github.rb

```diff
@@ -140,6 +140,7 @@ def open(url, data=nil)
     begin
       if data
         data_tmpfile.write data
+        data_tmpfile.close
```

Comment:
I think the only concern with `IO#flush` is that this doesn't guarantee the data is written to disk. That's what `IO#fsync` is for. But despite the buffering by the OS, the OS should be offering the same consistent view of the file if any process-specific buffers have been flushed.

But that's just for context and my understanding of these methods. It makes a lot of sense to close the file as soon as we're done writing to it, and I'd leave it as-is. I'm not sure either what the most idiomatic Ruby code is to do what is done here, but the change seems fine to me.